packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -649,10 +577,69 @@ $Calendar-dayPicker-margin: 10px;
   display: inline-block;
 }
 
-.navContainer {
-  @include float('right')
+.nextMonth,
+.nextYear {
+  margin-left: 3px;
+}
+
+.monthIsHighlighted{
+  background-color: $ms-color-themeLight;
+
+  &.monthOption:hover{
+    background-color: $ms-color-themeLight;
+  }
 }
 
 .monthIsCurrentMonth {
+  font-weight: 600;
+  color: $ms-color-white;
+  background-color: $ms-color-themePrimary;
+
+  &.monthOption:hover{
+    font-weight: 600;
+    color: $ms-color-white;
+    background-color: $ms-color-themePrimary;
+  }
+}
+
+.monthOption:active{
   background-color: $ms-color-themeLight;
 }
+
+
+//Rounded corners
+.roundedCornersTopLeft{
```

Comment:
Rename these classes to something more functional, i.e., selectedTopLeftCornerDate